joomla/joomla-cms
# [4.0] Change sort options to match columns order

### Testing Instructions
Go to Components > Smart Search > Content Maps
Click Sort Table By dropdown
See that sort options are in the same order as columns
